commands/shutdown.md

```diff
@@ -1,51 +1,74 @@
 * A replica with AOF enabled, reconnected with its master, performed a full resynchronization, and restarted the AOF file, triggering the initial AOF creation process. In this case not completing the AOF rewrite is dangerous because the latest dataset received from the master would be lost. The new master can actually be even a different instance (if the **REPLICAOF** or **SLAVEOF** command was used in order to reconfigure the replica), so it is important to finish the AOF rewrite and start with the correct data set representing the data set in memory when the server was terminated.
 
-There are conditions when we want just to terminate a Redis instance ASAP, regardless of what its content is. In such a case, the right combination of commands is to send a **CONFIG appendonly no** followed by a **SHUTDOWN NOSAVE**. The first command will turn off the AOF if needed, and will terminate the AOF rewriting child if there is one active. The second command will not have any problem to execute since the AOF is no longer enabled.
+There are situations when we want just to terminate a Redis instance ASAP, regardless of what its content is.
+In such a case, the command **SHUTDOWN NOW NOSAVE FORCE** can be used.
+In versions before 7.0, where the **NOW** and **FORCE** flags are not available, the right combination of commands is to send a **CONFIG appendonly no** followed by a **SHUTDOWN NOSAVE**.
+The first command will turn off the AOF if needed, and will terminate the AOF rewriting child if there is one active.
+The second command will not have any problem to execute since the AOF is no longer enabled.
+
+## Minimize the risk of data loss
+
+Since Redis 7.0, the server waits for lagging replicas up to a configurable `shutdown-timeout`, by default 10 seconds, before shutting down.
+This provides a best effort minimizing the risk of data loss in a situation where no save points are configured and AOF is disabled.
+Before version 7.0, shutting down a heavily loaded master node in a diskless setup was more likely to result in data loss.
+To minimize the risk of data loss in such setups, it's advised to trigger a manual `FAILOVER` (or `CLUSTER FAILOVER`) to demote the master to a replica and promote one of the replicas to be the new master, before shutting down a master node.
 
 @return
 
-@simple-string-reply on error.
-On success nothing is returned since the server quits and the connection is
-closed.
+@simple-string-reply: `OK` if `ABORT` was specified and shutdown was aborted.
+On successful shutdown, nothing is returned since the server quits and the connection is closed.
+On failure, an error is returned.
+
+@history
+
+* `>= 7.0.0`: `NOW`, `FORCE` and `ABORT` modifiers added.
+  Waiting for lagging replicas before exiting added.
```

Comment:
History is now recorded in the Redis repo, in each command's src/commands JSON file. Please move this tidbit there.

Reply:
ACK. I'll need to add it in the Redis PR then.